tests/foreman/ui/test_syncplan.py

```diff
@@ -643,8 +655,8 @@ def test_positive_synchronize_custom_products_future_sync_date(self):
             #  Verify products have not been synced yet
             for repo in repos:
                 self.validate_repo_content(
-                    repo.product.read().name,
-                    repo.name,
+                    repo.product,
```

Comment:
Have you tried running the test after update? I'm almost sure it will fail - repo.product is just initialized entity, it contains just `id`, no `name` or other fields, you have to `read()` it to obtain name. Without `read()` and no `name`, `self.validate_repo_content()` will fail as it tries to access `product.name`.
Same comment is applicable for few other `self.validate_repo_content()` calls too, please double check them.